components/GithubBadge.tsx

```diff
@@ -0,0 +1,10 @@
+const GithubBadge = () => {
+    return (
+      <div className="shadow-md cursor-pointer tracking-tight font-medium group mx-auto flex max-w-fit items-center justify-center space-x-2 overflow-hidden rounded-full border-2 bg-black hover:bg-black/90 px-7 py-1 border-black/20 hover:border-black text-white">
+        <p>✨ Star us on Github</p>
```

Comment:
Rather than text, add a logo with star image and number

Reply:
added the function to get repo stars, will soon update it with better design reference